.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           name: Invalidate Cloudfront
           command: |
             if [ "${CIRCLE_BRANCH}" == "production" ]; then
-              aws cloudfront create-invalidation --distribution-id ${PRODUCTION_CLOUDFRONT_ID} --paths /index.html /admin.html
```

Comment:
can you use `work` instead of `rebrand` ?

Reply:
👍